modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -276,6 +353,27 @@ private String _getAPIURL(
 		return _interpolateURL(sb.toString(), httpServletRequest);
 	}
 
+	private JSONObject _getFDSDateFilterJSONObject(
+		ObjectEntry fdsDateFilterObjectEntry) {
+
+		Map<String, Object> fdsDateFilterProperties =
+			fdsDateFilterObjectEntry.getProperties();
+
+		return JSONUtil.put(
+			"entityFieldType", fdsDateFilterProperties.get("type")
+		).put(
+			"id", fdsDateFilterProperties.get("fieldName")
+		).put(
+			"label", (String)fdsDateFilterProperties.get("label")
+		).put(
+			"max", fdsDateFilterProperties.get("to")
+		).put(
+			"min", fdsDateFilterProperties.get("from")
```

Comment:
According to [DateRangeFilter.js#L201-L205](https://github.com/liferay/liferay-portal/blob/8c68b43be493c0b7ee19908b4844a3c30655635a/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/DateRangeFilter.js#L201-L205), these values should be an object like `{day: '', month: '', year: ''}`

For reference: [DateRangeFDSFilterContextContributor.java#L44-L55](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-impl/src/main/java/com/liferay/frontend/data/set/internal/filter/DateRangeFDSFilterContextContributor.java#L44-L55)